js/jquery.js

```diff
@@ -0,0 +1,10364 @@
+/*!
+ * jQuery JavaScript Library v3.3.1
```

Comment:
use jquery as provided by owncloud core - please remove

Reply:
Understood, I removed it